pkg/config/config.go

```diff
@@ -0,0 +1,18 @@
+package config
+
+import (
```

Comment:
Maybe we move all configs here?

Reply:
I think it still make sense to have configs in their respective directories. Personally, I find it easier to read the code.